documentapi/src/vespa/documentapi/messagebus/policies/mirror_with_all.h

```diff
@@ -0,0 +1,27 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include <memory>
+
+class FastOS_ThreadPool;
+class FNET_Transport;
+class FRT_Supervisor;
+namespace slobrok { class ConfiguratorFactory; }
+namespace slobrok::api { class IMirrorAPI; }
+
+namespace documentapi {
+
+class MirrorAndStuff {
```

Comment:
Though I do enjoy this name greatly, I think it might be good to rename it to more accurately describe what kind of additional stuff it actually contains. The mind sparks with imagination and unlimited possibilities ✨🦄✨